integrations/influxdb_connector/influxdb-timestream-connector/template.yml

```diff
@@ -94,6 +94,10 @@ Parameters:
     MinValue: 2
     Default: 30000
     Description: The maximum amount of time in milliseconds an API Gateway event will wait before timing out.
+  RustLog:
+    Type: String
+    Default: 'INFO'
+    Description: The log level to use for the Lambda function. Typical values are error, warn, info, debug, trace, and off. Use trace in order to log the execution time of each function. Defaults to INFO.
```

Comment:
Worded as "Typical values are . . ." because there are more options for `RUST_LOG` than just the usual `error`, `warn`, etc.